src/components/sectionsForCartItemPage/sections.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../styles/utils/vars.scss";
+@import "../../styles/utils/mixins.scss";
+
+.grid-section-container {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  @include onDesktop1200 {
+    grid-template-columns: 1fr 1fr;
+    gap: 64px;
+  }
+}
+
+.about {
+  padding-bottom: 56px;
+
+  @include onTablet {
+    padding-bottom: 64px;
+  }
+
+  // @include onDesktop1200 {
+  //   display: grid;
+  //   grid-template-columns: 1fr 1fr;
+  // }
```

Comment:
What are these sus comments

Reply:
i deleted them, they are not necessary at all